src/lift-state-manager.ts

```diff
@@ -18,5 +18,146 @@ export default class LiftStateManager extends EventEmitter<Events> {
+    switch (motionState) {
+      case RomiCore.LiftState.MOTION_DOWN:
+        return 'Down';
+      case RomiCore.LiftState.MOTION_STOPPED:
+        return 'Stopped';
+      case RomiCore.LiftState.MOTION_UP:
+        return 'Up';
+      default:
+        return `Unknown (${motionState})`;
+    }
+  }
+
   private _liftStates: Record<string, RomiCore.LiftState> = {};
 }
+
+export class LiftRequestManager {
+  static liftModes = [
+    RomiCore.LiftRequest.REQUEST_END_SESSION,
+    RomiCore.LiftRequest.REQUEST_AGV_MODE,
+    RomiCore.LiftRequest.REQUEST_HUMAN_MODE,
+  ];
+
+  static doorModes = [
+    RomiCore.LiftRequest.DOOR_CLOSED,
+    RomiCore.LiftRequest.DOOR_OPEN,
+  ]
+
+
+  static getLiftRequestModes() {
+    let listOfModes: Array<any> = [];
+    this.liftModes.forEach(element => {
+      const key = this.liftModeToString(element);
+      const value = element;
+      listOfModes.push({ key, value })
+    });
+    return listOfModes
+  }
+
+  static getListOfLiftRequestModes() {
+    let listOfModes: Array<any> = [];
+    this.liftModes.forEach(element => {
+      listOfModes.push(this.liftModeToString(element))
+    });
+    return listOfModes
+  }
+
+  static getAllDoorModes() {
+
+    let listOfModes: Array<any> = [];
+    this.doorModes.forEach(element => {
+      const key = this.doorStateToString(element);
+      const value = element;
+      listOfModes.push({ key, value })
+    });
+    return listOfModes
+  }
+
+  static getListOfDoorModes() {
+    let listOfModes: Array<any> = [];
+    this.doorModes.forEach(element => {
+      listOfModes.push(this.doorStateToString(element))
+    });
+    return listOfModes
+  }
+
+  static liftModeToString(liftMode: number) {
+    switch (liftMode) {
+      case RomiCore.LiftRequest.REQUEST_END_SESSION:
+        return 'End Session';
+      case RomiCore.LiftRequest.REQUEST_AGV_MODE:
+        return 'AVG';
+      case RomiCore.LiftRequest.REQUEST_HUMAN_MODE:
+        return 'Human';
+      default:
+        return `Unknown (${liftMode})`;
+    }
+  }
+
+  static doorStateToString(doorState: number) {
+    switch (doorState) {
+      case RomiCore.LiftState.DOOR_OPEN:
+        return 'Open';
+      case RomiCore.LiftState.DOOR_CLOSED:
+        return 'Closed';
+      default:
+        return `Unknown (${doorState})`;
+    }
+  }
+
+  static StringToLiftMode(liftMode: string) {
+    switch (liftMode) {
+      case 'End Session':
+        return RomiCore.LiftRequest.REQUEST_END_SESSION;
+      case 'AVG':
+        return RomiCore.LiftRequest.REQUEST_AGV_MODE;
+      case 'Human':
+        return RomiCore.LiftRequest.REQUEST_HUMAN_MODE;
+      default:
+        return RomiCore.LiftRequest.REQUEST_END_SESSION;;
+    }
+  }
+
+  static StringToDoorState(doorState: string) {
+    switch (doorState) {
+      case 'Open':
+        return RomiCore.LiftState.DOOR_OPEN;
+      case 'Closed':
+        return RomiCore.LiftState.DOOR_CLOSED;
+    }
```

Comment:
I would avoid these functions, converting string back to number looks flaky, there could be many edge cases like caps, whitespaces, multi-language support that makes this unreliable. It seems its only used in the lift request form, it can receive the number and string in the props so it does not need to resort to this.